hcloud/actions/client.py

```diff
@@ -1,12 +1,21 @@
 # -*- coding: utf-8 -*-
-from hcloud.core.client import ClientEntityBase, BoundModelBase
+import time
 
-from hcloud.actions.domain import Action
+from hcloud.core.client import ClientEntityBase, BoundModelBase
+from hcloud.actions.domain import Action, ActionFailedException
 
 
 class BoundAction(BoundModelBase):
     model = Action
 
+    def wait_until_finished(self):
+        from hcloud import HcloudClient
+        while self.status == self.STATE_RUNNING:
+            self.reload()
+            time.sleep(HcloudClient.poll_interval)
```

Comment:
```suggestion
            time.sleep(poll_interval)
```
Do we use `poll_interval` anywhere else?  Consider moving it from `HcloudClient` to function argument

Reply:
On the hcloud-go we have it on the client too, because it could be used on more parts in the future. Even, when it is on one place you can change the `poll_interval` once instead of on every `wait_until_finished` call.

Reply:
In what other parts we can use `poll_interval`?

Reply:
Just imagine we would add a method to watch a specific server for changes. In my opinion, having it in a central place is the only solution we can use... Change the poll_interval once per client instead of once per `wait_until_finished` call.

Reply:
There are several problems with current implementation
1. You do an additional import inside the function.
2. You refer to the interval as `HcloudClient.poll_interval`.  Calling method `with_poll_interval` will not have any effects in this case.
3. Method `with_poll_interval` is not tested.
4. I see `poll_interval` as a part of `BoundAction`, not the client itself. If you can give me any examples how we can use the same variable for other features, I can think about better solution.

Reply:
If we are keeping `poll_interval` on the client level (to configure it once for all actions), I do not see why do we need `with_poll_interval` method. We can use `HcloudClient.__init__` to configure intervals.

Reply:
The idea was to have a fluent api (together with #9 ) for easily configure the client and have a similar configuration interface like hcloud-go.